frontend/public/locales/en/translation.json

```diff
@@ -404,8 +404,8 @@
     "clusterPool.creation.ocp.size": "Cluster pool size",
     "clusterPool.creation.success.message": "{{name}} was successfully created.",
     "clusterPool.creation.success.title": "Cluster pool created",
-    "clusterPool.creation.validation.unique.name": "This cluster pool name must be unique in this namespace: {{0}}.",
     "clusterPool.creation.validation.runningCount.lessThanOrEqualSize": "The cluster pool running count must be less than or equal to the size.",
+    "clusterPool.creation.validation.unique.name": "This cluster pool name must be unique in this namespace: {{0}}.",
```

Comment:
unrelated change but the release-2.5 is broken recently.
Let's rebase once fixed or commit right away now.